### DISCUSSION TITLE
Need help to control Meade ETX telescope AutoStar 497

### DISCUSSION THREAD

**Discussion by SixthGenie** [Stellarium/stellarium#2824] [Q&A]
Hi All,
I'm trying to control my telescope via Stellarium with no luck. 
I've downloaded the latest version 1.1 (1.22.4) , and setup the telescope as 

- Stellarium directly through serial port
- J2000
- Start/connect at startup

This is running on Windows 10.

Any attempt to Start telescope fails, with the following log in the log
_Creating telescope "Meade ETX105:TelescopeServerLx200:J2000:COM7:1200000" ; name/type/equinox/params: "Meade ETX105" "TelescopeServerLx200" J2000 "COM7:1200000"
TelescopeClientDirectLx200 parameters: port, time_delay: "COM7" 1200000
ERROR creating TelescopeClientDirectLx200: cannot open serial device "COM7"
TelescopeClient::create(): Unable to create a telescope client._

Serial port is USB at COM7, and I can control scope via the AutoStar Suite so I know communications should be possible. 

Any suggestions ?

**Comment by jhenley57** [comment]
I read some previous posts that suggested removing the 1.1 stellarium and falling back to the 0,22,xx something donr have it just right but the latest version avail that leads with a 0 "zero, having same problem here with my LX200 , works great with meade autostar but will not connect with stellarium 1.1.xx , going to try this myself as soon as I get the energy and the guts, worked on it 6 hrs yesterday.. hope this helps, looks like the 1.1 version wasnt finished baking I guess, JH :)

**Reply by gzotti** [reply]
1.1 has a problem fixed that indeed existed in 1.0. Some people have reported further issues with ASCOM telescopes, but neither LX200 nor Nexstar were so far reported to have issues in V1.1. The rest of your message is unfortunately unclear. Please write version numbers completely, not "0.22.something".

**Comment by jhenley57** [comment]
When you say you fixed the problem in 1.1 , I installed it yesterday, so how long has it been since you fixed, not working here?  Tnx JH

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Reply by gzotti** [reply]
I said I fixed _a_ problem. You did not specify yours in a comprehensive way. I will not further poke in possibility space for problems I cannot reproduce.

**Comment by jhenley57** [comment]
🙂
Thanks, no problem, is strickly hobby, appreciate the help.

I stopped doing astronomy for 20 years and just starting back, it will
take me a while to get back the knowledge I need, not changed much
just better, different software. Lots of new vendors for sure. 🙂

spent last 2 months remodeling and repairing the observatory building.
I should not have let it go so long without maint, but is nice again.

________________________________

**Comment by dogwhisper** [comment]
Was curious if this was ever figured out.  I'm having the same issue with 1.22.5 and a Meade ETX90ec with Autostar 497 using a USB to serial converter.  Been out of this for a few years too, but was hoping to use it for the comet the other night.  I have my instructions from 2019 that show just needing to go into the telescope control plugin and select the Meade 494 option.  Pretty straight forward com port stuff but it's not working any more.  I can connect to the scope through telnet and was able to update the Autostar to version 44Ec and can get it to work with the stelleriumscope with the Meade generic ASCOM drivers but stelleriumscope says it shouldn't be needed anymore with current Stellerium versions.   What am I missing?

**Reply by gzotti** [reply]
https://github.com/Stellarium/stellarium/wiki/Common-Problems-for-the-current-version#problems-with-telescopecontrol-plugin
